flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/SplitHelpers.java

```diff
@@ -62,8 +62,15 @@ public static List<IcebergSourceSplit> createSplitsFromTransientHadoopTable(
     final String warehouse = "file:" + warehouseFile;
     Configuration hadoopConf = new Configuration();
     final HadoopCatalog catalog = new HadoopCatalog(hadoopConf, warehouse);
+    ImmutableMap<String, String> properties = ImmutableMap.of(TableProperties.FORMAT_VERSION, "2");
```

Comment:
this hardcode the format to v2. what about adding a 2nd variation of `createSplitsFromTransientHadoopTable` with format version passed in?

Reply:
Since we are moving forward to make V2 the default, we might want to change the default to `2`, but kept it as `1` for now, since the goal of this PR is different